.github/workflows/full-ci.yml

```
@@ -0,0 +1,86 @@
+name: Full CI
+
+on:
+  push:
+    branches:
+      - 'main'
+    tags:
+      - 'v*.*.*'    
```

We could be more restrictive, similarly to what has been done in [powsybl-core CI](https://github.com/powsybl/powsybl-core/blob/4d431b1e0c157c05b41fd549ec844bbfc638cfe4/.github/workflows/maven.yml#L10)
```suggestion
      - 'v[0-9]+.[0-9]+.[0-9]+*'
```